src/controller/CHIPDevice.cpp

```diff
@@ -285,7 +285,7 @@ CHIP_ERROR Device::OpenPairingWindow(uint32_t timeout, PairingWindowOption optio
 
     ReturnErrorOnFailure(SendMessage(std::move(outBuffer), header));
 
-    setupPayload.version               = 1;
+    setupPayload.version               = 0;
```

Comment:
@pan-apple Can you please confirm if this is the right thing to do here as well?